log/field.go

```diff
@@ -143,6 +143,16 @@ func Object(key string, obj interface{}) Field {
 	}
 }
 
+// Event adds a string-valued field with name "value" to a Span.LogFields() record
+func Event(val string) Field {
+	return String("event", val)
+}
+
+// Message adds a string-valued field with name "event" to a Span.LogFields() record
```

Comment:
`Message creates a string-valued Field for span logs with key="message" and value=val.`